from0k2bp.tex

```diff
@@ -1048,26 +1062,25 @@
 step for the inner product
 proof}{The commitment step for the inner product proof}}\label{the-commitment-step-for-the-inner-product-proof}}
 
-The Prover P will need to send commitments to \emph{two } nonce vectors, analogous to the
+The Prover $\mathcal{P}$ will need to send commitments to \emph{two} nonce vectors, analogous to the
 $R$ value in the above description -- one for each of $\mathbf{x}$ and
-$\mathbf{y}$. These nonce vectors will be called $\mathbf{d}_x, \mathbf{d}_y$ respectively. But there's another
```

Comment:
I don't see a good reason to omit the `respectively`. It indicates to the reader something that is admittedly obvious, but there is no reason not to be precise - d_x is for x and d_y is for y.

Reply:
Added `respectively` back.